pkgs/development/python-modules/pysnooper/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, six
+, future
+, decorator
+}:
+
+buildPythonPackage rec {
+  version = "0.0.21";
+  pname = "PySnooper";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15vyc6hpy5whsswmz3a5qrcrcgd59lxf4q56krkcdzmk88agnwpg";
+  };
+
+  # Checks require python_toolbox, which has failing its own checks
+  # Issue reported upstream
+  doCheck = false;
+  propagatedBuildInputs = [
+    six
+    future
+    decorator
+  ];
+
+  meta = with stdenv.lib; {
```

Comment:
```suggestion
  meta = with lib; {
```

Reply:
I currently pushed this change. Sorry for my ignorance but when I change this and try to build with:
```
nix-build -I ~/devel/nixpkgs -E "with import <nixpkgs> {}; python3Packages.callPackage ~/devel/nixpkgs/pkgs/development/python-modules/pysnooper/default.nix { }"
```
I get: "error: undefined variable 'lib' at ..."
Am I missing something?

Reply:
Yes, you'll have to replace `stdenv` by `lib` in the arguments.

Reply:
Ah.. now the python_toolbox is required argument, but should be ok now. Thanks.